xrpl4j-core/src/main/java/org/xrpl/xrpl4j/model/flags/MpTokenIssuanceFlags.java

```diff
@@ -0,0 +1,151 @@
+package org.xrpl.xrpl4j.model.flags;
+
+/*-
+ * ========================LICENSE_START=================================
+ * xrpl4j :: core
+ * %%
+ * Copyright (C) 2020 - 2023 XRPL Foundation and its contributors
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =========================LICENSE_END==================================
+ */
+
+
+/**
+ * A set of static {@link Flags} which can be set on {@link org.xrpl.xrpl4j.model.ledger.MpTokenIssuanceObject}s.
+ */
+public class MpTokenIssuanceFlags extends Flags {
+
+  /**
+   * Constant for an unset flag.
+   */
+  public static final MpTokenIssuanceFlags UNSET = new MpTokenIssuanceFlags(0);
+
+  /**
+   * Constant {@link MpTokenIssuanceFlags} for the {@code lsfMPTLocked} account flag.
+   */
+  public static final MpTokenIssuanceFlags LOCKED = new MpTokenIssuanceFlags(0x00000001);
```

Comment:
Let's discuss this one in particular.

Reply:
"this one" meaning `LOCKED` or `MpTokenIssuanceFlags` having an `of(long)` method?